modules/runner/src/integrationTest/sql/filter/test_constant_comparisons.test_ignore

```diff
@@ -1,12 +0,0 @@
-# name: test/sql/filter/test_constant_comparisons.test_
-# description: Test expressions with constant comparisons
-# group: [filter]
-# Ignore: https://issues.apache.org/jira/browse/IGNITE-14617
```

Comment:
seems we need to close this issue as irrelevant ?

Reply:
Yes, but just after merge the PR